### PR TITLE
Changer MoCIGovernor

### PR DESCRIPTION
**Pull request by mulonemartin**

### CODE REVIEW

**Review by pmprete — APPROVED** [review]
👍

**Review by pmprete — APPROVED** [review]
👍

File: contracts/changers/governor/ProxyAdminIGovernorChanger.sol [hunk 1]
```diff
@@ -0,0 +1,25 @@
+pragma solidity 0.5.8;
```
Comment (pmprete) [inline contracts/changers/governor/ProxyAdminIGovernorChanger.sol:1]: We are using 0.5.17 now
```suggestion
pragma solidity ^0.5.8;
```

File: contracts/changers/governor/UpgradeDelegatorIGovernorChanger.sol [hunk 2]
```diff
@@ -0,0 +1,25 @@
+pragma solidity 0.5.8;
```
Comment (pmprete) [inline contracts/changers/governor/UpgradeDelegatorIGovernorChanger.sol:1]:
```suggestion
pragma solidity ^0.5.8;
```

**Review by pmprete — CHANGES_REQUESTED** [review]
Change solidity pragma

**Review by azboyzranch — APPROVED** [review]

**Pull request closed without merging**